tracing/src/main/java/org/jboss/resteasy/tracing/api/RESTEasyTracingInfoInterface.java

```diff
@@ -0,0 +1,9 @@
+package org.jboss.resteasy.tracing.api;
+
+public interface RESTEasyTracingInfoInterface {
```

Comment:
Are you actually using this interface? I'd say either you use it to reference the tracing info everywhere in the resteasy-jaxrs module, or you remove this interface at all.

Reply:
Unless I'm missing some usage of it ;-)

Reply:
yes the interface is needed by `RESTEasyTracingLoggerImpl`, which has this code:

```java
@Override
    public void flush(final MultivaluedMap<String, Object> headers) {
        final String[] messages = tracingInfo.getMessages();
        for (int i = 0; i < messages.length; i++) {
            headers.putSingle(String.format(RESTEasyTracing.HEADER_RESPONSE_FORMAT, i), messages[i]);
        }
    }
```

so the `getMessages` should be provided by concrete tracing info impl.

Reply:
let me move them into abstract class and remove interface.